src/helpers/formatters/file.js

```diff
@@ -23,25 +26,13 @@ export const tags = (tags, file) => {
   }
 }
 
-const formatImage = data => {
-  if (data) {
-    const imageFormat = data.data.format
-
-    const processImageByte = byte => {
-      return String.fromCharCode(byte)
-    }
-
-    const imageDecoded = data.data.data.map(
-      processImageByte
-    ).join('')
-
-    const imageBase64 = window.btoa(
-      imageDecoded
-    )
-
-    return [
-      `data:${imageFormat}`,
-      `base64,${imageBase64}`
-    ].join(';')
+/**
+ * Translate music-metadata picture object into a Data URI
+ * @param picture Cover
+ * @returns {string} Data URI
+ */
+function formatImage (picture) {
+  if (picture) {
```

Comment:
@Borewit `if` clause should be in `tags` function.

Reply:
That was not how is used to be.

Reply:
@Borewit I'm sorry. My mistake.
Please stick to [this one](https://github.com/staniel359/muffon/pull/36#discussion_r783864354).